### PR TITLE
Mostly formatting improvements, add wide ranging functionality (See description for more)

### PR DESCRIPTION
**Pull request by ricberht**
- Included greeting from the intergalactic federation via ASCII art.
- Console.log'd "~~~ wobble wobble wobble ~~~" at an optimal time.
- Added exclusionary conditions to single out the most evil people in the world in license.md.
- In a subtle but most importantly, legally careful way, makes implication that Bitcoin is part of a CIA conspiracy. I'm not saying look in to it but look in to it.
- Provide a quote relating to the dietary preferences of Sir Christopher Andrew Hoy, a former track cyclist and racing driver from Scotland who represented Great Britain at the Olympic and World Championships and Scotland at the Commonwealth Games.
- No further questions please. Unless they're requested changes.

### CODE REVIEW

**Comment by ricberht** [comment]
**_ Please note this will be radically easier to review if you turn off whitespace changes but if you don't wanna, go for it, knock yourself out _**

**Comment by ricberht** [comment]
Ready to review, i'm going to bed, I'll have a hang over tomorrow but will try to get any requested changes done then

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # These are supported funding model platforms
 
 github: Dishpit
```
Comment (Dishpit) [inline .github/FUNDING.yml:3]: i only have and use this currently; i've got others, but i don't use them much so the rest can stay blank for now

File: LICENSE.md [hunk 2]
```diff
@@ -16,8 +16,12 @@ The software may NOT be used by:
...
 - Specific prohibited entities, including but not limited to: Palantir Technologies, Northrop Grumman, Lockheed Martin, Raytheon, and OpenAI.
 - Any entity or individual engaging in activities deemed harmful, unethical, or exploitative, defined as:
-  - Actions that directly or indirectly cause physical, psychological, social, or environmental harm.
-  - Practices that violate widely accepted principles of human rights, equity, or sustainability, including but not limited to exploitation of vulnerable populations or destruction of natural habitats.
+    - Actions that directly or indirectly cause physical, psychological, social, or environmental harm.
+    - Practices that violate widely accepted principles of human rights, equity, or sustainability, including but not limited to exploitation of vulnerable populations or destruction of natural habitats.
+- Members of the klu Klux Klan or any other ethno-nationalist organisation
+- Members of any fascist organisation
+- Lawyers
+- Investment bankers
```
Comment (Dishpit) [inline LICENSE.md:25]: beautiful

**Review by Dishpit — CHANGES_REQUESTED** [review]

**Comment by Dishpit** [comment]
oh also merge conflicts lol

File: .github/FUNDING.yml (hunk 1 above)
Comment (ricberht) [reply .github/FUNDING.yml:3]: sorted

**Comment by ricberht** [comment]
Got that all sorted, also fixed the code inherited from merge conflicts, people forgot to close the script tags and all sorts so you had <style> inside script and all sorts of horrendous issues, all resolved. Hangover wasn't too bad after all :)

**Comment by Dishpit** [comment]
this seems to have a bug causing the cat spawner popup to show up infinitely

**Review by Dishpit — CHANGES_REQUESTED** [review]
prev. comment

**Pull request closed without merging**